cumulus/parachains/integration-tests/emulated/tests/bridges/bridge-hub-westend/src/tests/snowbridge_v2_inbound.rs

```diff
@@ -1092,3 +1101,240 @@ pub fn add_tip_from_asset_hub_user_origin() {
+
+		assert_expected_events!(
+			BridgeHubWestend,
+			vec![
+				RuntimeEvent::XcmpQueue(cumulus_pallet_xcmp_queue::Event::XcmpMessageSent { .. }) => {},
+				// Check that the relayer reward was registered.
+				RuntimeEvent::BridgeRelayers(pallet_bridge_relayers::Event::RewardRegistered { relayer, reward_kind, reward_balance }) => {
+					relayer: *relayer == relayer_account,
+					reward_kind: *reward_kind == BridgeReward::Snowbridge,
+					reward_balance: *reward_balance == relayer_reward,
+				},
+			]
+		);
+	});
+
+	AssetHubWestend::execute_with(|| {
+		type RuntimeEvent = <AssetHubWestend as Chain>::RuntimeEvent;
+
+		assert_expected_events!(
+			AssetHubWestend,
+			vec![
+				// message processed successfully
+				RuntimeEvent::MessageQueue(
+					pallet_message_queue::Event::Processed { success: true, .. }
+				) => {},
+			]
+		);
+
+		let events = AssetHubWestend::events();
+		println!("AssetHubWestend::events(): {:?}", events);
+	});
+
+	BridgeHubWestend::execute_with(|| {
+		type RuntimeEvent = <BridgeHubWestend as Chain>::RuntimeEvent;
+		assert_expected_events!(
+			BridgeHubWestend,
+			vec![RuntimeEvent::EthereumOutboundQueueV2(snowbridge_pallet_outbound_queue_v2::Event::MessageQueued{ .. }) => {},]
+		);
+	});
+}
+
+#[test]
+fn send_usdc_from_asset_hub_to_ethereum() {
+	fund_on_bh();
+	register_assets_on_ah();
+	fund_on_ah();
+
+	// Register USDC as a foreign asset
+	register_foreign_asset(usdc_location());
+
+	let snowbridge_sovereign = snowbridge_sovereign();
+
+	// Fund AssetHub sender with USDC
+	AssetHubWestend::execute_with(|| {
+		type RuntimeOrigin = <AssetHubWestend as Chain>::RuntimeOrigin;
+
+		// Mint USDC to sender
+		assert_ok!(<AssetHubWestend as AssetHubWestendPallet>::ForeignAssets::mint(
+			RuntimeOrigin::signed(snowbridge_sovereign),
+			usdc_location().clone().into(),
+			AssetHubWestendSender::get().into(),
+			32_864_721u128,
+		));
+	});
+
+	AssetHubWestend::execute_with(|| {
+		type RuntimeOrigin = <AssetHubWestend as Chain>::RuntimeOrigin;
+
+		// Build the XCM to send to BridgeHub, which will then export to Ethereum
+		let xcm = VersionedXcm::from(Xcm(vec![
+			UnpaidExecution { weight_limit: Unlimited, check_origin: None },
+			ExportMessage {
+				network: Ethereum { chain_id: 11155111 },
+				destination: Here,
+				xcm: Xcm(vec![
+					// Withdraw ETH for fees
+					WithdrawAsset(
+						vec![(Location::new(0, Here), 10_000_000_000_000_000u128).into()].into(),
+					),
+					// Pay fees with ETH
+					PayFees { asset: (Location::new(0, Here), 10_000_000_000_000_000u128).into() },
+					// Withdraw USDC
+					WithdrawAsset(
+						vec![(
+							Location::new(
+								0,
+								[AccountKey20 {
+									network: None,
+									key: hex!("a0b86991c6218b36c1d19d4a2e9eb0ce3606eb48"),
+								}],
+							),
+							32_864_721u128,
+						)
+							.into()]
+						.into(),
+					),
+					// Set origin to Ethereum beneficiary
+					AliasOrigin(Location::new(
```

Comment:
What happens if the InitiateTransfer does not forward origin. That should mean that the alias origin will not be here, rather it will be a clear origin, which will make it go down the v1 path and fail. I think the correct fix here is to change the way we detect the v1 path to look for PayFees instead of AliasOrigin.

https://github.com/paritytech/polkadot-sdk/blob/598feddb893f5ad3923a62e41a2f179b6e10c30c/bridges/snowbridge/primitives/outbound-queue/src/v2/converter/mod.rs#L120-L134